types/got/got-tests.ts

```diff
@@ -242,7 +243,7 @@ got('todomvc', {
 });
 
 got('todomvc', {
-    cache: new Map()
+    cache: new Keyv(),
```

Comment:
Would be nice to keep the `Map` test case as well, just to make sure that they both work